test/js/node/zlib/zlib.test.js

```diff
@@ -48,19 +59,126 @@ describe("zlib.gunzip", () => {
   });
 });
 
-describe("zlib.brotli*", () => {
-  it("returns stub", () => {
-    for (const method of [
-      "BrotliCompress",
-      "BrotliDecompress",
-      "brotliCompress",
-      "brotliCompressSync",
-      "brotliDecompress",
-      "brotliDecompressSync",
-      "createBrotliCompress",
-      "createBrotliDecompress",
-    ]) {
-      expect(() => zlib[method]()).toThrow(new Error(`zlib.${method} is not implemented`));
+describe("zlib.brotli", () => {
```

Comment:
Can you add a test that checks `node:http` can streamingly compress a body with brotli? via `.pipe`